test/framework/kubernetesversions/data/debian_injection_script.envsubst.sh.tpl

```diff
@@ -103,9 +103,9 @@ if [[ "$${KUBERNETES_VERSION}" != "" ]]; then
     systemctl restart kubelet
   fi
   IMAGE_REGISTRY_PREFIX=registry.k8s.io
```

Comment:
Please roll back changes in this file. We have to keep using the old registry for old Kubernetes versions